### PR TITLE
fix resource api  grpc failed to connect to unix socket

### PR DESCRIPTION
**Pull request by moshe010**
After commit https://github.com/k8snetworkplumbingwg/multus-cni/commit/c6fa047212efc3a410fb73aa83d2594a4f914320 resource api got broken in 2 places. first place handled by https://github.com/k8snetworkplumbingwg/multus-cni/commit/f530d3eb84608fa25a59737bcf49991d4a037d82 The second break was in replacing podresources.GetV1Client with getKubeletResourceClient. The GetV1Client remove the protocol (e.g unix:) by calling GetAddressAndDialer, but the getKubeletResourceClient is using the full endpoint (with the portocol) causing failed to connect to unix socket. This patch remove the unix: prefix before the grpc connect.

Fixes: #944
Signed-off-by: Moshe Levi <moshele@nvidia.com>

### CODE REVIEW

**Comment by moshe010** [comment]
@maiqueb  this should fix you issue. (tested on my local env)

File: pkg/kubeletclient/kubeletclient.go [hunk 1]
```diff
@@ -38,10 +39,14 @@ func LocalEndpoint(path, file string) (string, error) {
 	return filepath.Join(u.String(), file+".sock"), nil
 }
 
+func removeUnixProtocol(endpoint string) (string) {
+	return strings.TrimPrefix(endpoint, (unixProtocol + ":"))
+}
+
```
Comment (s1061123) [inline pkg/kubeletclient/kubeletclient.go:45]: How about to use `u, err := url.Parse(endpoint)` as `hasKubeletAPIEndpoint()` does?

```
func removeUnixProtocol(endpoint string) (string, error) {
    u, err := url.Parse(endpoint)
    if err != nil {
        return "", err
    }
   return u.Path, nil
}
```

**Comment by s1061123** [comment]
@maiqueb Hi, did you try the fix with your environment?

**Comment by maiqueb** [comment]
> @maiqueb Hi, did you try the fix with your environment?

Yes - it works as expected. Thank you @s1061123 / @moshe010 .

**Review by s1061123 — APPROVED** [review]
LGTM, merged.

**Pull request merged**